### DISCUSSION TITLE
Facing problems while making a PR

### DISCUSSION THREAD

**Discussion by harshal015** [open-sauced-craftwork/browser-extension#22] [Q&A]
Hello,

I have successfully developed an extension and I would like to submit it to GitHub. However, there are some important files inside the dist folder where I have written the code for the extension, but the dist folder is getting ignored because of the .gitignore file.

I am seeking guidance on how to proceed with uploading the extension to GitHub, taking into consideration the dist folder's inclusion. Should I remove the 'dist' from the .gitignore file and upload it?

Thank you for your time and assistance.

**Comment by takanome-dev** [comment]
Hey @Harshal2502, yes remove the `dist` from `.gitignore` and commit again.
Hopefully, that should solve your problem.

**Reply by harshal015** [reply]
Thanks, I just needed a confirmation 😄
